requirements.txt

```diff
@@ -0,0 +1,12 @@
+# Automatically generated by https://github.com/damnever/pigar.
+
+datasets==2.20.0
+huggingface-hub==0.23.4
+matplotlib==3.9.0
+mlx==0.15.1
```

Comment:
The model's behavior's been a bit weird with mlx 0.15.1 (released just a few days ago). I cannot yet pinpoint the exact cause, so for now, I am fixing the mlx version to 0.15.0 in this repository. Additionally, I am keeping the rest of the requirements.txt as is for now.